test/dummy-sorcery/config/initializers/sorcery.rb

```diff
@@ -0,0 +1,463 @@
+# The first thing you need to configure is which modules you need in your app.
```

Comment:
I missed this one so the `dummy-sorcery` app wasn't functioning 100% with my last PR, sorry!